config/print_target_properties.cmake

```diff
@@ -108,6 +108,6 @@ function(print_targets_properties)
   endforeach()
 endfunction()
 
-#------------------------------------------------------------------------------#
+#--------------------------------------------------------------------------------------------------#
 # End config/print_target_properties.cmake
-#------------------------------------------------------------------------------#
\ No newline at end of file
+#--------------------------------------------------------------------------------------------------#
```

Comment:
I'm not sure if this is important, but GitHub points out there is no newline here (and there wasn't before, either :D )

Reply:
I'm going to ignore that warning for now.  There are probably a lot of these issues in our source code.